### PR TITLE
docs: Improve API documentation - Pubkey

### PR DESCRIPTION
**Pull request by dubewarsagar**
* Considering only 1 endpoint (Pubkey) and 4 operation IDs under it (getPubkeyList,createPubkey,removePubkeyById,getPubkeyById)
* Removed duplicate information.
* Added detailed information of an endpoint at the top.
* Re-organised information to make the best use of space.

Feel free to suggest if there are any more modifications required.

Thanks for the contribution, make sure your commit message follows this subject style:

        type: brief summary up to 70 characters or
        type(scope): brief summary up to 70 characters

Type is required, scope is optional. Prefer lower-case and avoid dot at the end.

Find more info about types and scopes at:

https://github.com/RHEnVision/provisioning-backend#contributing

Take a moment to read our contributing and security guidelines:

https://github.com/RedHatInsights/secure-coding-checklist

**Checklist**

- [x] all commit messages follows the policy above
- [x] the change follows our security guidelines

### CODE REVIEW

File: api/openapi.gen.json [hunk 1]
```diff
@@ -1476,11 +1476,11 @@
     },
     "/pubkeys/{ID}": {
       "delete": {
-        "description": "A pubkey represents an SSH public portion of a key pair with name and body. If a pubkey was uploaded to one or more clouds, the deletion request will attempt to delete those SSH keys from all clouds. This means in order to delete a pubkey the account must have valid credentials to all cloud accounts the pubkey was uploaded to, otherwise the delete operation will fail and the pubkey will not be deleted from Provisioning database. This operation returns no body.\n",
+        "description": "Deletes SSH keys that were uploaded with the specified public key from all the clouds.\n",
```
Comment (ezr-ondrej) [inline api/openapi.gen.json:1479]: It deletes the key stored in our database AND deletes the uploaded ones, would it be clear to users from this? :thinking: 

Also we've removed the information about the possible failure, is that intentional?

File: api/openapi.gen.json [hunk 2]
```diff
@@ -1506,11 +1506,11 @@
         ]
       },
       "get": {
-        "description": "A pubkey represents an SSH public portion of a key pair with name and body. Pubkeys must have unique name and body (SSH public key fingerprint) per each account. Pubkey type is detected during create operation as well as fingerprints. Currently two types are supported: RSA and ssh-ed25519. Also, two fingerprint types are calculated: standard SHA fingerprint and legacy MD5 fingerprint available under fingerprint_legacy field. Fingerprints are used to check uniqueness of key.\n",
+        "description": "Gets details of the specified public key. \n",
```
Comment (ezr-ondrej) [inline api/openapi.gen.json:1509]: Nice :orange_heart:

File: api/openapi.gen.json (hunk 1 above)
Comment (dubewarsagar) [reply api/openapi.gen.json]: next commit fixes this issue. Information added back :)

**Comment by ezr-ondrej** [comment]
ok to test

**Review by ezr-ondrej — APPROVED** [review]
I'm wondering about the use of `Enter` it sounds bit weird in the API context, but what do I know! xD
Lets get that in! :+1:

**Comment by ezr-ondrej** [comment]
Thanks @dubewarsagar ! :orange_heart:

**Pull request merged**